pkg/lib/scoped/strategy.go

```diff
@@ -0,0 +1,132 @@
+package scoped
+
+import (
+	"errors"
+
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/scopedclient"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/rest"
+)
+
+var (
+	errGetterNotSpecified = errors.New("no getter func specified in order to retrieve a service account")
+)
+
+// NewStrategy returns a new instance of Strategy.
+func NewStrategy(logger *logrus.Logger, config *rest.Config, kubeclient operatorclient.ClientInterface, crclient versioned.Interface) *Strategy {
```

Comment:
Can we have a less generic name here (either in the package or in the actual names)?